From 3167336b5c41e2e81bc3ac117e447c90ec90ef0c Mon Sep 17 00:00:00 2001
From: "Luke D. Jones" <luke@ljones.dev>
Date: Sat, 13 Aug 2022 09:37:49 +1200
Subject: [PATCH 01/17] Fixes 98829e84dc67 ("asus-wmi: Add dgpu disable
 method")

The dgpu_disable attribute was not documented, this adds the
required documentation explaining how it works.

Signed-off-by: Luke D. Jones <luke@ljones.dev>
---
 Documentation/ABI/testing/sysfs-platform-asus-wmi | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/Documentation/ABI/testing/sysfs-platform-asus-wmi b/Documentation/ABI/testing/sysfs-platform-asus-wmi
index 04885738cf15..0f932fd60f4a 100644
--- a/Documentation/ABI/testing/sysfs-platform-asus-wmi
+++ b/Documentation/ABI/testing/sysfs-platform-asus-wmi
@@ -57,3 +57,12 @@ Description:
 			* 0 - default,
 			* 1 - overboost,
 			* 2 - silent
+
+What:          /sys/devices/platform/<platform>/dgpu_disable
+Date:          Aug 2022
+KernelVersion: 5.17
+Contact:       "Luke Jones" <luke@ljones.dev>
+Description:
+               Disable discrete GPU:
+                       * 0 - Enable dGPU,
+                       * 1 - Disable dGPU,
\ No newline at end of file
-- 
2.37.1

